tests/unit/dialog/core.js

```diff
@@ -75,6 +75,7 @@ QUnit.test( "ARIA", function( assert ) {
 	var element = $( "<div>" ).dialog(),
 		wrapper = element.dialog( "widget" );
 	assert.equal( wrapper.attr( "role" ), "dialog", "dialog role" );
+	assert.equal( wrapper.attr( "aria-modal" ), "true", "aria-modal attribute" );
```

Comment:
Please update line 73 from `assert.expect( 4 );` to `assert.expect( 5 );` as you added a test :-)

Reply:
ohhhhh that's what the number four in `assert.expect( 4 )` refers to, the number of tests. that makes sense, understood now. thanks! and just for the context, mentioned it on the issue, i am not a developer nor have i ever written a test. just tried to replicate the test pattern for one of the other attributes. pushed another commit updating to the suggested changes. setting the conversation to resolved